docs/standard/basetypes/alternation.md

```diff
@@ -0,0 +1,328 @@
+---
+title: Alternation constructs in regular expressions
+description: Alternation constructs in regular expressions
+keywords: .NET, .NET Core
+author: stevehoag
+manager: wpickett
+ms.date: 07/28/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: dotnet
+ms.assetid: 59ffac4d-fc6e-461f-8783-d9f8dc88ce2c
```

Comment:
ok, leave as-is. but just because we made them part of the standard now doesn't make them different. It's just that we updated the topic to be cross-platform now, no?